lms/services/organization_usage_report.py

```diff
@@ -30,6 +30,12 @@ class UsageReportRow:
     course_created: datetime
     authority_provided_id: str
 
+    @property
+    def is_teacher(self):
+        """Wether this row refers to a teacher."""
+        # Consider anyone for which we have an email a teacher
+        return self.email != "<STUDENT>"
```

Comment:
This is 100% true now, it might not be in the future.

Keep this as a property to allow changing the definition now.

Reply:
> Keep this as a property to allow changing the definition now.

Good idea